test/extended/olm/olmv1.go

```diff
@@ -97,6 +100,43 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLM
 			o.Expect(meta.IsStatusConditionPresentAndEqual(conditions, "Serving", metav1.ConditionTrue)).To(o.BeTrue())
 		}
 	})
+	g.It("should serve the /v1/api/all API endpoint", func(ctx g.SpecContext) {
+		checkFeatureCapability(ctx, oc)
+
+		catalogs := []string{
+			"openshift-certified-operators",
+			"openshift-community-operators",
+			"openshift-redhat-marketplace",
+			"openshift-redhat-operators",
+		}
+		// Start port-forward in background
+		_, _, _, err := oc.AsAdmin().Run("port-forward").Args("-n", "openshift-catalogd", "svc/catalogd-service", "8080:443").Background()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// Wait for port-forward to be established
+		time.Sleep(2 * time.Second)
+
+		for _, catalog := range catalogs {
+			g.By(fmt.Sprintf("Testing base API for catalog %q", catalog))
+			verifyAPIEndpoint(ctx, fmt.Sprintf("https://localhost:8080/catalogs/%s/api/v1/all", catalog))
+		}
+	})
+
+	g.It("[OCPFeatureGate:NewOLMCatalogdAPIV1Metas] should serve the /v1/api/metas API endpoint", func(ctx g.SpecContext) {
```

Comment:
Is this valid? Can `[OCPFeatureGate:NewOLMCatalogdAPIV1Metas]` be added to an It block?

Reply:
No, you should put it in it's own `g.Describe()`
Yes, you'll possibly duplicate a bunch of stuff, but that's OK.

Reply:
You can put each of these `g.It()` you've added as their own `g.Describe()` so they count as new tests... since you need to have 5.